NASA-PDS/pds-registry-mgr-elastic
# Update Schema Automatically

(1) See https://github.com/NASA-PDS/pds-registry-app/issues/146
(2) Java docs

**Test Instructions**
- Delete old registry: ```regfistry-manager delete-registry```
- Create new registry: ```regfistry-manager create-registry```
- Harvest some OREX data, for example ```https://sbnarchive.psi.edu/pds4/orex/orex.ovirs/data_raw/orbit_r/20191117T030457S365_ovr_scil0.xml```
- Load harvested OREX data: ```registry-manager load-data ...```
- Registry manager should automatically download OREX LDD and load harvested OREX data.
